packages/node_modules/@node-red/nodes/core/function/10-function.js

```diff
@@ -229,6 +233,7 @@ module.exports = function(RED) {
         }
         var context = vm.createContext(sandbox);
         try {
+            vm.runInContext(iniText, context);
```

Comment:
Does this need to dedicated error handling so we can tell the user where any errors are in the code? See the `vm.createScript` call a few lines below where we set `displayErrors` and `filename`. Need to make sure we provide a consistent (and good!) experience when errors occur.

I also suggest we skip the call entirely if there in no initialise function code provided - no point invoking the vm if there is nothing to do.

Reply:
Added error handling as per your advice.